1-exercises/I-string-replace/exercise.js

```diff
@@ -10,10 +10,13 @@
   "I like cats. One night I went to the park and I saw 100000 cats. It was a brilliant night."
 */
 
-let story =
-  "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";
-
 let result = story.replace("", "");
```

Comment:
Please paste here original story and check your output

`let story =
  "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";`